openstudiocore/src/bimserver/BIMserverConnection.cpp

```diff
@@ -49,6 +49,13 @@
 #include <iostream>
 #include <boost/none.hpp>
 
+///
+/// TODO: Remove this helper when Qt is fully removed
+///
+static auto toQString(const std::string &s) {
```

Comment:
@lefticus I didn't mention but bimserver project is not being de-qt-ized.  It is going with the other graphical components.  I'll change this to use the "../model_editor/Utilities.hpp" file

Reply:
I wondered about it, but didn't see it mentioned in any of the previous emails, I might have missed it (t's also not a project I'm familiar with)

Well, at least I did this as a tiny patch here, making the fixes much easier!